src/WebPush.php

```diff
@@ -165,27 +178,27 @@ public function flush($batchSize = null)
                     /** @var RequestException $reason **/
                     $reason = $result['reason'];
 
-                    $error = array(
+                    $error = [
                         'success' => false,
-                        'endpoint' => "".$reason->getRequest()->getUri(),
+                        'endpoint' => $reason->getRequest()->getUri(),
```

Comment:
`getUri(). __toString()` ?

You could add another `endpointUri` key which gives `getUri()` only ([UriInterface](https://github.com/php-fig/http-message/blob/master/src/UriInterface.php)), but I'm not sure about the use case. I feel like only a string is sufficient